File: examples-sdk/typescript/201-llm-basic-chat.ts
#!/usr/bin/env tsx
// 201-llm-basic-chat — TypeScript SDK equivalent
// Compile: vil compile --from typescript --input 201-llm-basic-chat.ts --release

import { VilPipeline, VilServer, ServiceProcess } from "vil-sdk";

const server = new VilServer("llm-basic-chat", 3100);
const chat = new ServiceProcess("chat");
chat.endpoint("POST", "/chat", "chat_handler");
server.service(chat);
server.compile();

File: examples-sdk/python/201-llm-basic-chat.py
#!/usr/bin/env python3
"""201-llm-basic-chat — Python SDK equivalent
Compile: vil compile --from python --input 201-llm-basic-chat.py --release
"""
import os
from vil import VilPipeline, VilServer, ServiceProcess

server = VilServer("llm-basic-chat", port=3100)
chat = server.service_process("chat")
chat.endpoint("POST", "/chat", "chat_handler")
server.compile()
